test/lib/ansible_test/_util/target/setup/bootstrap.sh

```diff
@@ -337,18 +321,11 @@ bootstrap_remote_rhel_9()
             ${py_pkg_prefix}-cryptography
             ${py_pkg_prefix}-pyyaml
             "
-
-        # The following OS packages are missing for 3.11 (and possibly later) so we just
-        # skip them and let ansible-test install them from PyPI.
-        if [ "${python_version}" = "3.9" ]; then
-            packages="
-                ${packages}
-                ${py_pkg_prefix}-packaging
-                ${py_pkg_prefix}-resolvelib
-            "
-        fi
     fi
 
+    # packaging and resolvelib are missing for Python 3.11 (and possible later) so we just
+    # skip them and let ansible-test install them from PyPI.
+
```

Comment:
I'd move this up just below the comment about Jinja2, since that's where the list of packages for the controller are defined.